### ISSUE TITLE
README example does not work

### ISSUE THREAD

**Issue by LMS007** [patriksimek/vm2#391]
I'm trying to allow required modules to work and when I follow the example, I get an error

``` js
try {
  const {NodeVM} = require("vm2");
  const vm = new NodeVM({
    require: {
      external: true,
    }
  });
  vm.run("var request = require('request');");
  } catch(e) {
    console.error("ERROR ", e.message);
  }
```
    
OUTPUT: 
` ERROR  You must specify script path to load relative modules.`

I'm running node v14 and vm2 v3.9.5

This is almost exactly the example in the readme so is must be broken?

This seems strange because the module here "request" does not start with a relative path or a dot.
https://github.com/patriksimek/vm2/blob/3a21c36222bc6b8e7948068c9152f9d691fd52de/lib/sandbox.js#L318

how does that regex pass?

**Comment by XmiliaH** [comment]
You need to supply the filename as second argument to `run` as the example does. If this is not done the module is not associated with a directory and relative loads of modules will cause the observed error.

**Comment by LMS007** [comment]
Thanks. I realized the line of code I referenced was not a passing condition, it was going into the else and then the `currentDirname` is null so that check fails. Are you saying I just need to pass a file name is?

I think this is the bit was useful to read
https://github.com/patriksimek/vm2#loading-modules-by-relative-path

It was just a lot further down in the README, so It's unclear why the filepath param is needed since every example that follows the first didnt use it.
